sw/device/lib/testing/test_framework/ottf_isrs.c

```diff
@@ -10,6 +10,15 @@
 #include "sw/device/lib/runtime/ibex.h"
 #include "sw/device/lib/runtime/log.h"
 
+/**
+ * These weak symbols enable the OTTF ISRs to be used without the OTTF itself.
+ * This enables us to maintain one set of default ISRs for testing, while also
+ * enabling writing tests that do not make use of the OTTF. See the `crt_test`
+ * in `sw/device/tests/crt_test.c` for an example.
+ */
```

Comment:
Maybe moving this paragraph just above `ottf_exception_handler`, would be better? Otherwise we are talking about the ISRs but bundling in `pxCurrentTCB` and `kTestConfig`?

Reply:
Sorry, this comment only applies to the weak pxCurrentTCB and kTestConfig symbols. Specifically, since the `ottf_isrs.S` assembly requires these symbols be defined, if we don't define them somewhere and try to use the OTTF ISRs lib **_without_** the  main OTTF lib (as in the case in the `crt_test`) then we will get a link error, since these symbols are only defined in the OTTF lib (not the OTTF ISR lib, these are two separate libs). The work-around for this is to declare them as weak symbols in the OTTF ISRs lib, and let the OTTF lib override them. I updated the comment to make it clear it only applies to the pxCurrentTCB and kTestConfig symbols. Does that make sense?